src/v2/guide/events.md

```diff
@@ -223,24 +223,28 @@ The `.passive` modifier is especially useful for improving performance on mobile
 
 ## Key Modifiers
 
-When listening for keyboard events, we often need to check for common key codes. Vue also allows adding key modifiers for `v-on` when listening for key events:
+When listening for keyboard events, we often need to check for specific keys. Vue allows adding key modifiers for `v-on` when listening for key events:
 
 ``` html
-<!-- only call `vm.submit()` when the `keyCode` is 13 -->
-<input v-on:keyup.13="submit">
+<!-- only call `vm.submit()` when the `key` is PageDown -->
+<input @keyup.page-down="onPageDown">
```

Comment:
Sorry for late, why here the `key` becomes `PageDown` not still `Enter`? (The `keyCode` of `Enter` is `13` and the `keyCode` of `PageDown` is `34`. Ref: https://keycode.info, https://css-tricks.com/snippets/javascript/javascript-keycodes/)
I think it's better to remain the key `Enter` is this case to match the meaning of "submit".

Thanks.

Reply:
That's a good point @Jinjiang, sorry I missed this. @kokopelli314 would you mind a subsequent PR?